contrib/ci/Jenkinsfile

```diff
@@ -0,0 +1,27 @@
+pipeline {
+    agent { label 'gluon-docker' }
+    environment {
+        GLUON_SITEDIR = "contrib/ci/minimal-site"
+        GLUON_TARGET = "x86-64"
+        BUILD_LOG = "1"
+    }
+    stages {
+        stage('lint') {
+            steps {
+                sh 'luacheck --std=min {packages,scripts,target}'
+            }
+        }
+        stage('docs') {
+            steps {
+                sh' make -C docs html'
```

Comment:
It will be good if the docs can be deployed as well.

Reply:
That certainly would be awesome, but I'd say lets get this going first before we learn to fly :)

Reply:
I remember ffrgb/meshviewer was using https://www.netlify.com/ for test deployments.